hal/targets/hal/TARGET_ARM_SSG/TARGET_BEETLE/mbed_sdk_init.c

```diff
@@ -23,8 +23,10 @@ void mbed_sdk_init(void) {
     EFlash_DriverInitialize();
     EFlash_ClockConfig();
 
+#ifndef FEATURE_UVISOR
```

Comment:
@fvincenzo If `FCache` APIs are stateless (i.e. they do not rely on zeroed or initialized data) you can keep the 3 function calls below enabled and move them to before uVisor.

You can either move this code to the end of `SystemInit`, or add a specific hook just for them:

```asm
ldr r0, =SystemInit
blx r0
ldr r0, =fcache_hook
blx r0
ldr r0, =uvisor_init
blx r0
...
```

```C
void fcache_hook(void)
{
    /* Enable Flash Cache Stats */
    FCache_DriverInitialize();
    FCache_Enable(1);
    FCache_Invalidate();
}
```

Note: The stateless requirement is important as uVisor will reset the C/C++ lib memories.

@meriac